docs/tokens.md

```diff
@@ -30,11 +36,19 @@ TxData {
 }
 ```
 
-## Issue Tokens
-When issuing a new token, we specify the data for creating a new token in the transaction input, where the `token_id` is a hash of the inputs. **TODO which inputs?**
-**TODO explain remaining fields**
+## Issuing Tokens
+
+**TODO is the sentence below true? In the example, I don't see any data relating to the token in the input.** 
+
+When issuing a new token, we specify the data for creating a new token in the transaction input, where the `token_id` is a hash of the inputs.
+
+**TODO explain the remaining fields (token_ticker, metadata_URI)**
```

Comment:
`token_ticker` is a short name of the token. No longer than 5 symbols.

`metadata_URI` is a web link to a JSON file where we can store additional information for the token: https://github.com/mintlayer/core/pull/67 Over here you can find the paragraph: `Format of data located by reference metadata_URI`

Reply:
@sinitcin Which characters can be used in the token ticker? Are numbers, special characters allowed?